generator/templates/schema.gotmpl

```diff
@@ -90,6 +90,19 @@ func unmarshal{{ pascalize .Name }}(data []byte, consumer runtime.Consumer) ({{
 
 }
 {{ else }}{{ if or .IsComplexObject .IsTuple .IsAdditionalProperties }}{{ if .Name }}type {{ if not .IsExported }}{{ .Name }}{{ else }}{{ pascalize .Name }}{{ end }} {{ end }}{{ template "schemaBody" . }}
+{{if .Default}}
+func ({{.ReceiverName}} *{{ pascalize .Name }}) UnmarshalJSON(b []byte) error {
+  type {{ pascalize .Name }}Alias {{ pascalize .Name }}
+  var t {{ pascalize .Name }}Alias
+  if err := json.Unmarshal([]byte(`{{json .Default}}`), &t); err != nil {
```

Comment:
It's safer to use: `[]byte({{ printf "%q" (json .Default) }})`
Otherwise if you have a string property in the json that contains a backtick it won't compile